### PR TITLE
Add documentation to help new users

### PR DESCRIPTION
**Pull request by louisfelix**
Hi!
This PR is a small README documentation contribution.
The proposed modifications are the following:

- Indent headers to sub-headers for better reading experience (except the first header)
- Add an "Install" sub-heading to the README (as seen in other exporters: useful hint if a user want to install the exporter but is not familiar with Go)
- Adding "Usage" sub-heading to the README (as seen in other exporters; useful in analysis phases to quickly understand usage without installing and running --help command)
- Change "./Azure-metrics-exporter" to "./azure-metrics-exporter" to
reflect default binary name
- Add an "Exporter configuration" sub-heading to re-group corresponding
section and inform on the default "azure.yml" (this is not obvious for a new user)
- Add an "Azure account requirements" under "Exporter Configuration" (useful clarification if a user want to configure the exporter but is not familiar with Azure credentials).

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,19 +6,48 @@ Azure metrics exporter for [Prometheus.](https://prometheus.io)
...
+
+```bash
+go get -u github.com/RobustPerception/azure_metrics_exporter
+```
+
+## Usage
+```bash
+azure_metrics_exporter [<flags>]
+
+Flags:
```
Comment (brian-brazil) [inline README.md]: I'd prefer not to have two places to update things.

Comment (louisfelix) [reply README.md]: I understand your point, this is for sure a drawback of my approach.
What do you think if I keep the "Usage" section, with this line only 
`azure_metrics_exporter --help`

Comment (brian-brazil) [reply README.md]: Sure

**Comment by brian-brazil** [comment]
Thanks!

**Comment by louisfelix** [comment]
Thanks to you @brian-brazil for your quick reply!

**Pull request merged**